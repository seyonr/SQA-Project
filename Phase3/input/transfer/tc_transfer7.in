login 
admin
12345
T
99999
Bill Gates
00000
1200
No
L