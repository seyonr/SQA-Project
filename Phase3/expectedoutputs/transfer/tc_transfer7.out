Welcome to the banking system!
Do you want to login or exit : 
Enter login type (admin or standard): 
Enter account number: 
Login successful!

What do you want to do today?
W – Withdrawal
T - Transfer
P - Paybill
D - Deposit
C - Create
DE - Delete
DI - Disable
CH - Change Plan
L – Logout
Please enter the account number you want to transfer from : 
Please enter the account name you want to transfer from : 
Please enter the account number you want to transfer to : 
How much would you like to transfer : 
Transfer successful for backend
Transfer successful, would you like to to make another transfer : 

What do you want to do today?
W – Withdrawal
T - Transfer
P - Paybill
D - Deposit
C - Create
DE - Delete
DI - Disable
CH - Change Plan
L – Logout
Logged out!
Have a great day, see you soon!
